server/lorax_server/utils/paged_attention.py

```diff
@@ -49,6 +49,7 @@ def attention(
     block_tables: torch.Tensor,
     input_lengths: torch.Tensor,
     max_s: int,
+    num_kv_heads: int,
```

Comment:
I'm confused, this should have been fixed by https://github.com/predibase/lorax/commit/8454a8240d3f88baf6220399cfcd18137a5e04bd#diff-ed2c9bc1d24333087be66a6001624aab40c9ea17076cd23d21d306399fe5ce95

It seems the comparison is out of date with main?